### DISCUSSION TITLE
Tailwind 4 Support

### DISCUSSION THREAD

**Discussion by blowsie** [RyanClementsHax/tailwindcss-themer#138] [General]
With Tailwind 4 nearing its release, have any preparations been made to make this plugin compatible? 
https://tailwindcss.com/blog/tailwindcss-v4-alpha

**Comment by RyanClementsHax** [comment]
Yeah I'm waiting for v4 to stabilize and the evaluate what needs to happen for v4 support.

It seems with the removal of the config, much of this may be invalidated so I just want to wait to be extra sure.

**Reply by joshuajung** [reply]
Hi @RyanClementsHax, just checking in to ask if you have any plans to upgrade themer, now that v4 is out. I truly enjoy working with it, thanks for your efforts!

**Reply by RyanClementsHax** [reply]
Yup! I just haven't gotten around to it yet.

**Reply by azharuddin-mulla-amla** [reply]
Hi @RyanClementsHax, we early waiting for themer with tailwind v4
